src/api/types/generated-schema.ts

```diff
@@ -3295,6 +3301,16 @@ export interface components {
       /** Format: int32 */
       totalPlants: number;
     };
+    PlantingSiteValidationProblemPayload: {
```

Comment:
This isn't part of this work, is it? I suppose it is OK to leave in here since it is probably needed by a subsequent PR and it isn't changing or removing anything.

Reply:
Happy to take that out if the convention in the FE code is to hand-edit the generated types rather than just taking the latest schema.

Reply:
I think in this case it is fine since it is additive. Generally if I see unrelated changes I will do a patch add and confirm the chunks I want. No change needed here.